tests: mem protection tests updated tags

maksimmasalski opened the pull request:
Update current tests Doxygen tags to make them more informative for tests:
-test_mslab()
-test_create_alt_thread()
-test_sys_heap_mem_pool_assign()

For QA is very important to understand goal of each test. I think, adding informative Doxygen tags to the test will help developers to understand Zephyr testing system better.

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>

Commit:
tests: mem protection tests updated tags

Updated current tests tags to make them more informative.
1. test_mslab updated Doxygen tag
2. test_create_alt_thread updated Doxygen tag
3. test_sys_heap_mem_pool_assign updated Doxygen tag

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>

maksimmasalski commented:
@nashif @andyross @andrewboie There is no serious code logic changes, please review changes in Doxygen comments.

dcpleung commented on tests/kernel/mem_heap/mheap_api_concept/src/test_mheap_concept.c:
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Intel Corporation
+ * Copyright (c) 2020 Intel Corporation
```
Don't replace the year in copyright, but add to it. For example, `Copyright (c) 2016, 2020 Intel Corporation`.

maksimmasalski replied:
Understood, I didn't know that.

andyross reviewed (commented):
Might consider calling out "Doxygen links" or something in the title.  "test tags" usually refers to the YAML syntax in test cases used by filtering rules.

maksimmasalski commented:
> Might consider calling out "Doxygen links" or something in the title. "test tags" usually refers to the YAML syntax in test cases used by filtering rules.

Thanks, I will keep in mind.

Commit:
tests: copyright message add year to the existing

Instead of replacing of copyright year with the new one,
necessary to add new to the existing one

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>

dcpleung reviewed (dismissed):
The other copyright headers also need to be added, and not replaced.

Commit:
tests: copyright message add year to the existing

Instead of replacing of copyright year with the new one,
necessary to add new to the existing one

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>
